src/harness/harness.ts

```diff
@@ -2068,7 +2072,7 @@ namespace Harness {
                 for (let {done, value} = gen.next(); !done; { done, value } = gen.next()) {
                     const [name, content, count] = value as [string, string, number | undefined];
                     if (count === 0) continue; // Allow error reporter to skip writing files without errors
-                    const relativeFileName = relativeFileBase + (ts.startsWith(name, "/") ? "" : "/") + name + extension;

```

Comment:
Do we now call `sanitizeTestFilePath` on `relativeFileName` somewhere?

Reply:
We did before - the paths given to this function are always sanitized. What was changed was stripping the leading slash during sanitization, rather than here.